src/App.tsx

```diff
@@ -30,7 +31,10 @@ const guestRoutes: Array<GuestRouteInterface> = [
   { path: "/binnentuin/menu", component: MenuPageBinnentuin, exact: true },
   { path: "/inloggen", component: Login, exact: true },
   { path: "/register", component: Register, exact: true },
-  { path: "/winkelmand", component: WinkelmandPage, exact: true },
+
+  { path: "/winkelmand", component: WinkelmandPage, exact: true},
+  { path: "/menuRoof", component: TheRoofPage, exact: true},
```

Comment:
```suggestion
  { path: "/theroof/menu", component: TheRoofPage, exact: true},
```

Om in het zelfde stramien te blijven, misschien deze url gebruiken?

Reply:
Jaa logisch!